eureka/src/main/java/com/linecorp/armeria/server/eureka/EurekaUpdatingListener.java

```diff
@@ -328,26 +330,37 @@ public void run() {
             client.sendHeartBeat(appName, instanceId, instanceInfo, null)
                   .aggregate()
                   .handle((res, cause) -> {
-                      try (HttpData content = res.content()) {
-                          if (closed) {
-                              return null;
-                          }
+                      if (closed) {
+                          return null;
+                      }
```

Comment:
Could you wrap this entire block with try-finally and call `res.content().close()` in the `finally` block when `res` is not `null`, so that we do not leak anything?

Reply:
We don't use `aggregateWithPooledObjects` anymore so we might not need that? 🤔

Reply:
Ah, then OK as it is.